### DISCUSSION TITLE
Connect vactubes

### DISCUSSION THREAD

**Discussion by Badpoolinator** [BEEmod/BEE2-items#3769] [Tech Support / Q&A] [answered]
Is there a way to connect vactubes? It's really annoying not being able to, so please tell me if it's possible.

**Comment by FrozenDragon0** [comment, marked as answer]
What do you mean you can't connect them? You right click on them and connect them to another Vactube item.

**Reply by Stnkybo** [reply]
I’ve gotten it to work but it can be a pain.
You have to only place one single vac tube for every turn or straight line and connect them with ant lines on the order you connect them is important

**Reply by FrozenDragon0** [reply]
Yes I am fully aware as to how they work. It's not a great system but there is not much we can do about that.
